src/chrome/content/rules/Senate.gov.xml

```diff
@@ -15,7 +15,10 @@
 
 		<!-- The rules subdomain is all 403 via TLS -->
 		<!-- As is the Senate Intelligence Committee -->
-		<exclusion pattern="^http://(?:www.)?(?:intelligence|rules)\.senate\.gov/" />
+		<exclusion pattern="^http://(?:www.)?intelligence\.senate\.gov/" />
+		<exclusion pattern="^http://(?:www.)?rules\.senate\.gov/" />
```

Comment:
The style guide talks about "long strings of subdomains" as those can be expressed via `<target host />` instead. It doesn't apply to exclusion pattern, especially for two subdomains.